web/templates/post/index.html.eex

```diff
@@ -5,6 +5,25 @@
       <%= markdown String.slice(post.content, 0, 325) <> "..." %>
     </p>
     <a href="/posts/<%= post.id %>" class="pv2 ph2 f5 f4-l lh-copy">Read this story</a>
+    <%= if @current_user do %>
```

Comment:
Seems pretty similar to what's in `web/templates/tip/index.html.eex` is there an easy way to DRY it up?